packages/devtools_app/lib/src/info/flutter/info_screen.dart

```diff
@@ -124,17 +125,30 @@ class _FlagList extends StatelessWidget {
               mainAxisAlignment: MainAxisAlignment.spaceBetween,
               children: <Widget>[
                 Expanded(
-                  child: DefaultTaggedText(
-                    '<bold>${flag.name}</bold>\n'
-                    '${flag.comment}',
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Text(flag.name, style: bold(defaultTextTheme)),
+                      Text(flag.comment),
+                    ],
                   ),
                 ),
-                Container(
+                ConstrainedBox(
                   constraints: const BoxConstraints(minWidth: 100.0),
-                  child: DefaultTaggedText(
-                    '<primary-color>${flag.valueAsString}</primary-color>\n'
-                    '<primary-color-light>$modifiedStatusText</primary-color-light>',
-                    textAlign: TextAlign.end,
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.end,
+                    children: [
+                      Text(
```

Comment:
this change may have also fixed how this view renders with CanvasKit. The dependency on DefaultTaggedText may have been causing problems with the latest CanvasKit version. Fyi @hterkelsen